bin/clang/pragmas.cc

```diff
@@ -644,69 +657,36 @@ SSTGlobalVariablePragma::activate(Decl *d)
   errorAbort(d, "global pragma should only be applied to statements");
 }
 
-SSTNullVariablePragma::SSTNullVariablePragma(SourceLocation loc, const std::list<Token> &tokens)
+SSTNullVariablePragma::SSTNullVariablePragma(SourceLocation /**loc*/, std::map<std::string, std::list<std::string>>&& args)
```

Comment:
Since you changed this type, any reason for it to be a list? Why not a vector or SmallVector?

Reply:
This was the type used by other ArgMap pragmas. We can change these to optimized small vectors on a later PR.